flask_rebar/rebar.py

```diff
@@ -459,9 +461,23 @@ def add_handler(
         :param Type[USE_DEFAULT]|None|str mimetype:
             Content-Type header to add to the response schema
         """
-        if isinstance(response_body_schema, marshmallow.Schema):
+        if isinstance(response_body_schema, marshmallow.Schema) or (
+            isclass(response_body_schema)
+            and issubclass(response_body_schema, marshmallow.Schema)
+        ):
```

Comment:
Any reason not to just use `isinstance(response_body_schema, dict)`?

Reply:
Was just going to propose something similar! The more generic way to write that would be:
```python
if not isinstance(response_body_schema, collections.abc.Mapping):
    ...
```
The only issue is if the API is meant to support passing in a generator of `(key, value)` pairs as well as some kind of `Mapping`, but probably fair to just document that this case only supports passing in a `dict` and use `if not isinstance(response_body_schema, dict):`

Reply:
Also, what do you think of moving this whole check inside the `if response_body_schema:` check below? That would better consolidate all the handling of `response_body_schema` in one place.